extensions/sql-database-projects/src/models/tree/projectTreeItem.ts

```diff
@@ -24,13 +26,15 @@ export class ProjectRootTreeItem extends BaseProjectTreeItem {
 
 		this.project = project;
 		this.dataSourceNode = new DataSourcesTreeItem(this);
+		this.databaseReferencesNode = new DatabaseReferencesTreeItem(this);
 
 		this.construct();
 	}
 
 	public get children(): BaseProjectTreeItem[] {
 		const output: BaseProjectTreeItem[] = [];
 		output.push(this.dataSourceNode);
+		output.push(this.databaseReferencesNode);
```

Comment:
What's SSDT's behavior here: does it always show up, or only if there are references defined?